server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -245,68 +245,7 @@ public void testEmptyExtensionsFile() throws Exception {
         expectThrows(IOException.class, () -> new ExtensionsOrchestrator(settings, extensionDir));
     }
 
-    public void testExtensionsInitialize() throws Exception {
```

Comment:
Do you just want to do `@Ignore` for now? [1]

[1] https://www.guru99.com/junit-ignore-test.html

Reply:
This is better than removing the test.
You can ignore it like mentioned below and add the issue to it.

```
 @Ignore("<Issue Link>")

Reply:
The only problem with this solution is that the tests as is passes on local machines, but not in Jenkins.  So by changing the expectations to "Unseen", the PR gradle check would pass but after that the local gradle checks would fail for everyone.

Reply:
I think if I just comment out the assert statements and add a similar message to yours, then the same functionality that you are suggesting could be implemented.

Reply:
its nice to see we have checks every where :)